scripts/python/read_dense.py

```diff
@@ -29,7 +31,10 @@
 #
 # Author: Johannes L. Schoenberger (jsch-at-demuc-dot-de)
 
+import argparse
 import numpy as np
+import os
+import pylab as plt
```

Comment:
This should stay in the main function, otherwise matplotlib becomes a hard dependency when importing this file from somewhere else.

Reply:
Is this file going to be imported anywhere else? I thought it is meant for visualization purposes, not part of a library.

Reply:
I use this file in my personal scripts for parsing dense outputs.